content/how-to-write-code/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Chapter 3: How to Write Code"
+date: 2023-07-13T14:00:03-05:00
+draft: false
+weight: 3
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: Lin Wang # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Learning Objectives
+
+Upon completing all the content in this chapter, you should be able to do the following:
+
+1. Write `console.log` statements
+1. Understand what syntax, comments and debugging are when you are coding
+
+## Key Terminology
+
+Here is a list of the key terms throughout this chapter based on the page they first appear on. You should make note of these.
+
```

Comment:
The Key Terminology is blank for now. 

Also the structure of table of contents still looks off to me